virtualization/windowscontainers/container-networking/network-drivers-topologies.md

```diff
@@ -43,13 +43,6 @@ In addition to leveraging the default 'nat' network created by Docker on Windows
 
   In configuration 2 users will need to add a endpoint on the host network compartment that acts as a gateway and configure routing capabilities for the designated prefix.
   > Requires: Requires Windows Server 2016, Windows 10 Creators Update, or a later release.
```

Comment:
Can you remove the text from this line #45 as well. It is not needed because containers are not supported on any OS lower than RS1.

Reply:
@vidushv  If you can make this change, since it's your branch, it will be easier. You can then click Resolve conversation. After that, I'll go ahead and approve/publish.

Reply:
Awesome, just made the change. Thanks for the review!